### PR TITLE
Pet bugs

### PR DESCRIPTION
**Pull request by tmccoy529**
17032: Adoptable Pets - Missing Data on Cards: Arrival Date, Species, Weight

17047: Adoptable Pets - Image Missing Alt Text; ID Label Capitalization

Weight already had a section for it like Age so once the service is sends weight and weight unit back through this should pick it back up.

I also build alt text for images if we dont return that value. At one point we did return that value so again like weight if its added back it should pick it back up otherwise we will generate it on the fly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,5 +37,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```
Comment (martypowell) [inline package.json:41]: this can be removed for now, sorry about that.

File: src/components/PetAttributeRows.jsx [hunk 2]
```diff
@@ -10,21 +10,39 @@ import PetAttributes from "./PetAttributes";
...
+    "en-US",
+    options
+  );
 
   return (
     <Table className="table-fixed">
       <TableBody>
         <TableRow key={animalId}>
-          <TableHeadCell>Animal Id</TableHeadCell>
+          <TableHeadCell>Animal ID</TableHeadCell>
```
Comment (martypowell) [inline src/components/PetAttributeRows.jsx:30]: Was this intentional?

File: src/components/PetThumbnail.jsx [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
+      style={{ maxWidth: "100%" }}
+      src={thumbnail}
+      alt={
+        thumbnailAltText
+          ? thumbnailAltText
+          : `This is ${animalName}, this ${gender} is adoptable `
+      }
+    />
   ) : (
     <i className="far fa-newspaper fa-3x dg_news_icon"></i>
```
Comment (martypowell) [inline src/components/PetThumbnail.jsx:15]: this is a newspaper? this should probalby be updated?

File: src/components/PetAttributeRows.jsx (hunk 2 above)
Comment (tmccoy529) [reply src/components/PetAttributeRows.jsx:30]: Yes Jason said that in prod the ID is all capital and to change it to match

File: src/components/PetThumbnail.jsx (hunk 3 above)
Comment (tmccoy529) [reply src/components/PetThumbnail.jsx:15]: done

**Pull request merged**